cli/cmd/lint.go

```diff
@@ -13,8 +13,8 @@ import (
 func (r *runners) InitLint(parent *cobra.Command) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "lint",
-		Short: "Lint Helm charts",
-		Long:  `Lint Helm charts defined in .replicated config file. This command reads chart paths from the .replicated config and executes helm lint locally on each chart.`,
+		Short: "Lint Helm charts and Preflight specs",
+		Long:  `Lint Helm charts and Preflight specs defined in .replicated config file. This command reads paths from the .replicated config and executes linting locally on each resource.`,
```

Comment:
```suggestion
		Long:  `Lint Helm charts and Preflight specs defined in .replicated config file. This command reads paths from the .replicated config and executes linting locally on each resource.`,
		SilenceUsage: true,
```

When the linter finds a syntax error, it does error out with a non zero code which is good, but it also adds the --help and flags at the end like if it were a malformed command rather than an intentional error, adding this would still make it error out but not add 
```
Usage:
  replicated lint [flags]

Flags:
  -h, --help   help for lint

Global Flags:
      --app string     The app slug or app id to use in all calls
      --debug          Enable debug output
      --token string   The API token to use to access your app in the Vendor API
```
at the end